src/components/modal-confirm/modal-confirm.jsx

```diff
@@ -41,6 +41,16 @@ var ModalConfirm = React.createClass({
     });
   },
   render: function () {
+    var buttons = this.state.buttons.map((button, i) => {
```

Comment:
Yep, there is basically 1 instance of the modal used throughout the app, so `fire` includes the button metadata needed.

https://github.com/mozilla/webmaker-core/pull/789/files#diff-dcf4652e72bbc422d5d5834d3015f951R73